res/css/views/dialogs/_MessageEditHistoryDialog.scss

```diff
@@ -59,10 +59,10 @@ limitations under the License.
         }
 
         .mx_EventTile_line {
```

Comment:
This block could be moved to `_EventTile.scss` by setting `mx_MessageEditHistoryDialog_edits &`, but that should not be applied yet before analysis because that way `_EventTile.scss` would possibly become much larger than it is, when you apply the same policy to other files such as `_TimelineCard.scss`.